File: launchdarkly/ruby/src/UpdateFlagLinkExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

patch_operation_1 = LaunchDarklyClient::PatchOperation.new
patch_operation_1.op = "replace"
patch_operation_1.path = "/title"

patch_operation = [
    patch_operation_1,
]

begin
    response = LaunchDarklyClient::FlagLinksBetaApi.new.update_flag_link(
        "projectKey_string", # project_key
        "featureFlagKey_string", # feature_flag_key
        "id_string", # id
        patch_operation,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling FlagLinksBetaApi#update_flag_link: #{e}"
end

File: launchdarkly/php/src/UpdateFlagLinkExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

$patch_operation_1 = (new LaunchDarkly\Client\Model\PatchOperation())
    ->setOp("replace")
    ->setPath("/title");

$patch_operation = [
    $patch_operation_1,
];

try {
    $response = (new LaunchDarkly\Client\Api\FlagLinksBetaApi(config: $config))->updateFlagLink(
        project_key: "projectKey_string",
        feature_flag_key: "featureFlagKey_string",
        id: "id_string",
        patch_operation: $patch_operation,
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling FlagLinksBetaApi#updateFlagLink: {$e->getMessage()}";
}
